tests/sentry/uptime/consumers/test_results_consumer.py

```diff
@@ -636,9 +638,10 @@ def test_onboarding_failure(self):
         )
         with (
             mock.patch("sentry.quotas.backend.remove_seat") as mock_remove_seat,
-            mock.patch("sentry.uptime.consumers.results_consumer.metrics") as metrics,
+            mock.patch("sentry.uptime.consumers.results_consumer.metrics") as consumer_metrics,
+            mock.patch("sentry.uptime.detectors.result_handler.metrics") as onboarding_metrics,
             mock.patch(
-                "sentry.uptime.consumers.results_consumer.ONBOARDING_FAILURE_THRESHOLD", new=2
+                "sentry.uptime.detectors.result_handler.ONBOARDING_FAILURE_THRESHOLD", new=2
```

Comment:
Ideally we split out some of the tests to test the functionality of the onboarding result handler separately